plasma_framework/test/helpers/transaction.js

```diff
@@ -0,0 +1,51 @@
+const rlp = require('rlp');
+
+const NULL_DATA = Buffer.alloc(32, 0);
+const TransactionTypes = {
+  PLASMA_DEPOSIT: 1
+}
+
+class TransactionOutput {
+    constructor(amount, owner, token) {
+        this.amount = amount;
+        this.outputGuard = owner;
+        this.token = token;
+    }
+
+    formatForRlpEncoding() {
+        return [this.amount, this.outputGuard, this.token]
+    }
+}
+
+class Transaction {
```

Comment:
Probably we want to rename this as `PaymentXXX` as well

Reply:
done